<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Motion Simulator User Interface</string>
  </property>
  <widget class="QGraphicsView" name="qGraphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelActuatorControls">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>200</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Actuator Controls:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSequences">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>200</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Sequences:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAct1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>180</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Actuator 1 Angle:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAct2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>305</y>
     <width>180</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Actuator 2 Angle:</string>
   </property>
  </widget>
  <widget class="QSlider" name="angle2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>325</y>
     <width>140</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-45</number>
   </property>
   <property name="maximum">
    <number>45</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelAct3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>180</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Actuator 3 Angle:</string>
   </property>
  </widget>
  <widget class="QSlider" name="angle3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>400</y>
     <width>140</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>198</width>
      <height>198</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
  </widget>
  <widget class="QPushButton" name="homeButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>190</y>
     <width>175</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset Actuator Angles</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>290</y>
     <width>175</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable All Motors</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>390</y>
     <width>175</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>E-Stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>800</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Motion Simulator Design Team</string>
   </property>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="angle1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>18</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
